css/style.css

```diff
@@ -0,0 +1,403 @@
+html,
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+ul {
+  list-style: none;
+  display: block;
+  margin-block-start: 0;
+  margin-block-end: 0;
+  margin-inline-start: 0;
+  margin-inline-end: 0;
+  padding-inline-start: 0;
+}
+
+figure {
+  display: block;
+  margin-block-start: 0;
+  margin-block-end: 0;
+  margin-inline-start: 0;
+  margin-inline-end: 0;
+}
+
+body {
+  top: 50px;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
+
+img {
+  max-width: 100%;
+}
+
+nav {
+  border-bottom: 1px solid #e2e2e2;
+  background: white;
+  width: 100vw;
+  display: table;
+  padding-bottom: 0.25rem;
+  position: fixed;
+  top: 0;
+  left: 0;
+}
+
+nav .left {
+  float: left;
+  padding: 0.5rem 1rem;
+}
+
+nav .right {
+  float: right;
+  padding-right: 1rem;
+}
+
+nav .icon {
+  height: 1.15rem;
+  margin-right: 0.75rem;
+}
+
+nav .logo {
+  margin-left: 25%;
+}
+
+nav .logo img {
+  height: 1.5rem;
+  margin-top: 0.45rem;
+}
+
+nav .page-name {
+  font-weight: bold;
+  font-family: "Libre Franklin", sans-serif;
+  text-transform: uppercase;
+}
+
+nav .right button {
+  padding: 0.45rem 1rem;
+  margin: 0.25rem 0.25rem;
+  font-size: 11px;
+  font-weight: 500;
+  color: white;
+  text-transform: uppercase;
+  background-color: #6288a5;
+  border: 1px solid #326891;
+  border-radius: 4px;
+}
+
+header {
+  padding: 10vh 12.5vw 0;
+}
+
+.main-title {
+  padding: 0 10vw 2rem;
+  font-size: 2.5rem;
+  font-style: italic;
+  line-height: 1.25;
+  font-family: Georgia, "Times New Roman", Times, serif;
+}
+
+.img-caption {
+  font-size: 0.8rem;
+  margin-top: 0.25rem;
+  padding-right: 20%;
+  color: #666;
+}
+
+.img-caption span {
+  color: #999;
+}
+
+main {
+  padding: 0.7rem 12.5vw;
+}
+
+.article-heading {
+  padding-left: 25%;
+  padding-bottom: 2.5rem;
+}
+
+.article-heading .article-author {
+  font-weight: bold;
+  font-size: 0.85rem;
+}
+
+.article-heading .date-and-social {
+  /* vertical-align: middle; */
```

Comment:
#### `REQUIRED`

Consider deleting commented out codes. If you need to review a previous version of your code, use git history.